src/utils/DepositUtils.ts

```diff
@@ -125,6 +127,73 @@ export async function queryHistoricalDepositForFill(
   };
 }
 
+export async function findDeposit(
```

Comment:
Also RE https://github.com/across-protocol/sdk/pull/798#issuecomment-2549449045, I think this will make it more difficult to handle an SVM integration, because the caller (or this function) will need to consider whether the target chain is evm or svm.

So in general my preference would be to route calls via the SpokePoolClient, and have it implement pointers to the correct underlying interfaces depending on whether it's evm or svm.

Reply:
hm i'm ok with that